go/client/cmd_device_remove.go

```diff
@@ -134,7 +138,11 @@ func NewCmdDeviceRemove(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 		Flags: []cli.Flag{
 			cli.BoolFlag{
 				Name:  "f, force",
```

Comment:
Maybe `--force` should be renamed, given that it does not actually force the removal in the "last device" case?